CONTRIBUTING.md

```diff
@@ -15,10 +14,17 @@ Thank you for your interest in contributing to the Uniswap interface! 🦄
 1. `yarn install`
 1. `yarn build`
 
+## Running tests
+
+1. Run unit tests: `yarn test`
+1. Run unit tests and show test coverage info for all tests: `yarn test --coverage --watchAll`
+1. Run unit tests and show test coverage info only for tests affected by your changes: `yarn test --coverage --watch`
```

Comment:
This should be the standard command. Is there any reason to not include `--coverage` in `yarn test`?
Also, `--watch` is done by default through jest, and does not need to be included.